app/validators/password_integrity_validator.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+class PasswordIntegrityValidator < ActiveModel::EachValidator
+  def self.is_strong_enough?(password, record)
+    # Scoring of the test is a range of 0 - 4
+    #   0: too guessable: risky password. (guesses < 10^3)
+    #   1: very guessable: protection from throttled online attacks. (guesses < 10^6)
+    #   2: somewhat guessable: protection from unthrottled online attacks. (guesses < 10^8)
+    #   3: safely unguessable: moderate protection from offline slow-hash scenario. (guesses < 10^10)
+    #   4: very unguessable: strong protection from offline slow-hash scenario. (guesses >= 10^10)
+    # (from https://github.com/dropbox/zxcvbn#usage)
```

Comment:
Instead of the long comment, we recommend:

```
# zxcvbn scores go from 0-4; see docs: https://github.com/dropbox/zxcvbn#usage
```

Reply:
travis thinks this is fine and cool as is